packages/@aws-cdk/aws-timestream/package.json

```diff
@@ -81,25 +81,38 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@aws-cdk/cdk-integ-tools": "0.0.0",
     "@aws-cdk/assertions": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-sns": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/core": "0.0.0",
+    "constructs": "^10.0.0",
     "@aws-cdk/cdk-build-tools": "0.0.0",
     "@aws-cdk/cfn2ts": "0.0.0",
     "@aws-cdk/pkglint": "0.0.0",
     "@types/jest": "^27.4.1"
   },
   "dependencies": {
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^10.0.0"
   },
   "peerDependencies": {
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-sns": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^10.0.0"
   },
   "engines": {
     "node": ">= 14.15.0"
   },
-  "stability": "experimental",
-  "maturity": "cfn-only",
+  "stability": "stable",
```

Comment:
Please don't declare this to be stable immediately. It should start out as `experimental/experimental`.